src/index.js

```diff
@@ -13,23 +13,26 @@ program
 
 program.parse(process.argv);
 
-const simulationData = init({
-    populationSize: config.POPULATION_SIZE,
-    spreadRadius: config.SPREAD_RADIUS,
-    hygieneDisregard: config.HYGIENE_DISREGARD,
-    hospitalEffectiveness: config.HOSPITAL_EFFECTIVENESS,
-    hospitalCapacity: config.HOSPITAL_CAPACITY,
-    populationPreset: config.POPULATION_PRESET,
-    incubationPeriod: config.INCUBATION_PERIOD,
-    infectionPeriod: config.INFECTION_PERIOD,
-    quarantineEffectiveness: config.QUARANTINE_EFFECTIVENESS,
-    quarantinePeriod: config.QUARANTINE_PERIOD,
-    quarantineDelay: config.QUARANTINE_DELAY,
-    quarantineType: config.QUARANTINE_TYPE,
-    quarantinePercentage: config.QUARANTINE_PERCENTAGE,
-    testRate: config.TEST_RATE,
-    testCooldown: config.TEST_COOLDOWN,
-});
+console.log(config.SIMULATION_PRESET);
+const simulationData = init(
+    (config.SIMULATION_PRESET) ? { ...SimulationPresets[config.SIMULATION_PRESET] } :
+        {
+            populationSize: config.POPULATION_SIZE,
+            spreadRadius: config.SPREAD_RADIUS,
+            hygieneDisregard: config.HYGIENE_DISREGARD,
+            hospitalEffectiveness: config.HOSPITAL_EFFECTIVENESS,
+            hospitalCapacity: config.HOSPITAL_CAPACITY,
+            populationPreset: config.POPULATION_PRESET,
+            incubationPeriod: config.INCUBATION_PERIOD,
+            infectionPeriod: config.INFECTION_PERIOD,
+            quarantineEffectiveness: config.QUARANTINE_EFFECTIVENESS,
+            quarantinePeriod: config.QUARANTINE_PERIOD,
+            quarantineDelay: config.QUARANTINE_DELAY,
+            quarantineType: config.QUARANTINE_TYPE,
+            quarantinePercentage: config.QUARANTINE_PERCENTAGE,
+            testRate: config.TEST_RATE,
+            testCooldown: config.TEST_COOLDOWN,
+        });
```

Comment:
Please declare the object before, and pass it to init, so it becomes clearer about what's going on

Reply:
Done! :)